cross/tvheadend/Makefile

```diff
@@ -11,7 +11,12 @@ PKG_DIR = $(PKG_NAME)-$(PKG_GIT_HASH)
 DEPENDS  = cross/openssl cross/gnutls
 DEPENDS += cross/uriparser cross/pcre2 cross/libiconv cross/fontconfig
 DEPENDS += cross/libhdhomerun cross/libdvbcsa cross/dvb-apps cross/pngquant
+DEPENDS += cross/libwebsockets
+# To build curl with zstd feature define depends zstd before cross/curl
+DEPENDS += cross/zstd
 DEPENDS += cross/curl
+# curl build needs additional configuration for gnutls and libssh2
+export BUILD_CURL_WITH="gnutls libssh2"
```

Comment:
While curl only has a `--with-zstd=PATH` and `--without-zstd`, this could also be added as a `BUILD_CURL_WITH` flag.